release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,71 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+- **TODO**
+
+### Kubernetes v1.16.3
+
+- No change from the previous version
+
+### Calico v3.10.1
+
+- No change from the previous version
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+
+- No change from the previous version
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
```

Comment:
@axbarsan legacy cluster-operator v0.22.1 has been released

Reply:
Thank you @sslavic